File: js/과일장수.js
function solution(k, m, score) {
    var answer = 0;
    var arr = [];
    score.sort((a,b) => b-a);
    for (let i=0; i<score.length; i+=m) {
        arr.push(score.slice(i, i+m));
    }
    for (let j=0; j<arr.length; j++) {
        if(m === arr[j].length) {
            answer += Math.min(...arr[j]) * m;       
        }
    }
    return answer;
}

File: python/과일장수.py
def solution(k, m, score):
    answer = 0
    arr = []
    score.sort(reverse=True)
    for i in range(0, len(score), m):
        arr.append(score[i:i+m])
    for j in range(len(arr)):
        if len(arr[j]) == m:
            answer += min(arr[j]) * m
    return answer